pkg/apis/maistra/v1/servicemeshmemberroll_types.go

```diff
@@ -58,17 +61,114 @@ type ServiceMeshMemberRollSpec struct {
 	// +optional
 	// +nullable
 	Members []string `json:"members,omitempty"`
+
+	// Include namespaces with label keys and values matching this selector.
+	// +optional
+	MemberSelector *metav1.LabelSelector `json:"memberSelector,omitempty"`
+
+	// List of namespaces that should be excluded, even if matched by a wildcard
+	// in spec.members or by the memberSelector.
+	// +optional
+	ExcludeNamespaces []string `json:"excludeNamespaces,omitempty"`
+}
+
+func (smmr *ServiceMeshMemberRoll) IsWildcard(ns string) bool {
```

Comment:
We need to allow wildcards like `kube-*` in `excludeNamespaces`. So, to keep things consistent, we allow them also in `members` (instead of only allowing `*` there).